app/frontend/src/components/Navigation/ExternalNavButton.tsx

```diff
@@ -0,0 +1,41 @@
+import { Typography, TypographyProps } from "@material-ui/core";
+import makeStyles from "utils/makeStyles";
+
+const useStyles = makeStyles((theme) => ({
+  link: {
+    color: theme.palette.text.secondary,
+    display: "flex",
+    flex: "1",
+    fontSize: "2rem",
+    maxWidth: theme.spacing(21),
+    padding: theme.spacing(1, 1.5),
+    transition: theme.transitions.create(["color", "padding-top"], {
+      duration: theme.transitions.duration.short,
+    }),
```

Comment:
What happens if we actually remove this? 🤔 
I got a feeling this is left over from when we had the old nav link style before so might be unnecessary now

Reply:
You're right! Nothing changes without it 🙂